target/linux/mediatek/filogic/base-files/etc/board.d/01_leds

```diff
@@ -29,6 +29,12 @@ xiaomi,redmi-router-ax6000-stock|\
 xiaomi,redmi-router-ax6000-ubootmod)
 	ucidef_set_led_netdev "wan" "wan" "rgb:network" "wan"
 	;;
+zbtlink,zbt-z8103ax)
+	ucidef_set_led_netdev "lan1" "lan1" "green:status" "lan1" "link tx rx"
+	ucidef_set_led_netdev "lan2" "lan2" "green:status" "lan2" "link tx rx"
+	ucidef_set_led_netdev "lan3" "lan3" "green:status" "lan3" "link tx rx"
```

Comment:
Does this (3 devices using the same LED) really work at expected? I'd expect the last one to prevail.

Reply:
As idea: to bind this led to "br-lan" instead of the lan ports. @darknblack what is indicating led in stock firmware?

Reply:
Hi, it turns out, I don't have to add lan1-3 in the file. It seems like the led will automatically light up when you insert an rj45 to the port.

Also, I have added wlan2g and wlan5g. Thank you :)